ecology.yaml

```diff
@@ -0,0 +1,414 @@
+    owner: iuc
+    tool_panel_section_label: Graph/Display Data
+
+  - name: ggplot2_violin
+    owner: iuc
+    tool_panel_section_label: Graph/Display Data
+
+  - name: ggplot2_heatmap2
+    owner: iuc
+    tool_panel_section_label: Graph/Display Data
+
+  - name: ggplot2_point
+    owner: iuc
+    tool_panel_section_label: Graph/Display Data
+
+  - name: ggplot2_histogram
+    owner: iuc
+    tool_panel_section_label: Graph/Display Data
+
+  - name: tsne
+    owner: iuc
+    tool_panel_section_label: Graph/Display Data
+
+  - name: best_regression_subsets
+    owner: devteam
+    tool_panel_section_label: Statistics
+
+  - name: principal_component_analysis
+    owner: devteam
+    tool_panel_section_label: Statistics
+
+  - name: kernel_principal_component_analysis
+    owner: devteam
+    tool_panel_section_label: Statistics
+
+  - name: kernel_canonical_correlation_analysis
+    owner: devteam
+    tool_panel_section_label: Statistics
+
+  - name: canonical_correlation_analysis
+    owner: devteam
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_svm_classifier
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_sample_generator
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_searchcv
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_build_pipeline
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_pairwise_metrics
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_nn_classifier
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_numeric_clustering
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_ensemble
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_feature_selection
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_data_preprocess
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_clf_metrics
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_regression_metrics
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_discriminant_classifier
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_generalized_linear
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_model_validation
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+  - name: scipy_sparse
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+  - name: sklearn_suite
+    owner: bgruening
+    tool_panel_section_label: Statistics
+
+
+
+
+  - name: csv_to_tabular
+    owner: mnhn65mo
+    tool_panel_section_label: File Conversion
+    
+
+  - name: tabular_to_csv
+    owner: mnhn65mo
+    tool_panel_section_label: File Conversion
+    
+
+  - name: butterfly_analysis
```

Comment:
Do you think the tools from `mnhn65mo` are ready for review? Or do you need more time?

Reply:
I don't think butterfly_analysis is on github?

Reply:
Concerning butterfly_analysis, I think this is the last PR we are waiting from @alanamosse... Tools will be ready for review at the end of the day.